packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -53,7 +51,12 @@ export class OverflowSetBase extends BaseComponent<IOverflowSetProps, {}> implem
     }
 
     return (
-      <Tag aria-orientation={vertical ? 'vertical' : 'horizontal'} {...uniqueComponentProps} className={this._classNames.root} role={role}>
+      <Tag
+        role={role || 'group'}
+        aria-orientation={role === 'menubar' ? (vertical === true ? 'vertical' : 'horizontal') : undefined}
```

Comment:
`role === 'menu' || role === 'menubar` here ?

Reply:
Overflow set probably won't be used for a role="menu" (which is usually a contextual menu), so menubar is the primary role we're accounting for.